### PR TITLE
Feature/ju 25 java 21 mnycz

### PR DESCRIPTION
**Pull request by mnycz-ch**
https://companieshouse.atlassian.net/browse/JU-25

### CODE REVIEW

**Review by foman-ch — APPROVED** [review]

**Review by akunati-ch — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -182,16 +203,35 @@
...
+        <dependency>
+            <groupId>uk.org.webcompere</groupId>
+            <artifactId>system-stubs-core</artifactId>
+            <version>2.1.6</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>uk.org.webcompere</groupId>
+            <artifactId>system-stubs-jupiter</artifactId>
+            <version>2.1.6</version>
```
Comment (hwatkins-ch) [inline pom.xml:218]: Put these versions into a property

File: pom.xml [hunk 2]
```diff
@@ -182,16 +203,35 @@
...
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.github.docker-java</groupId>
+            <artifactId>docker-java-api</artifactId>
+            <version>3.3.6</version>
```
Comment (hwatkins-ch) [inline pom.xml:224]: Put version into a property

**Review by hwatkins-ch — CHANGES_REQUESTED** [review]
Looks good, just a couple versions should go into properties

File: pom.xml (hunk 1 above)
Comment (mnycz-ch) [reply pom.xml:218]: done

**Pull request merged**